archetypes/client/src/main/resources/archetype-resources/src/main/java/Application.java

```diff
@@ -0,0 +1,155 @@
+package ${package};
+
+import java.util.Arrays;
+import java.util.concurrent.CompletableFuture;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.annotation.ClientCacheEntryCreated;
+import org.infinispan.client.hotrod.annotation.ClientCacheEntryModified;
+import org.infinispan.client.hotrod.annotation.ClientCacheEntryRemoved;
+import org.infinispan.client.hotrod.annotation.ClientListener;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.client.hotrod.event.ClientCacheEntryCreatedEvent;
+import org.infinispan.client.hotrod.event.ClientCacheEntryModifiedEvent;
+import org.infinispan.client.hotrod.event.ClientCacheEntryRemovedEvent;
+
+/**
+ * Sample client application code. For more examples please see our client documentation
+ * (https://infinispan.org/docs/stable/titles/hotrod_java/hotrod_java.html).
+ * <p>
+ * In order to run this application, it's necessary for a Infinispan server to be executing on your localhost with the
+ * HotRod endpoint available on port 11222. The easiest way to achieve this, is by utilising our server docker image:
+ * <p>
+ * <code>docker run --net=host -p 11222:11222 -e USER="user" -e PASS="pass" infinispan/server</code>
```

Comment:
for some reason, `--net=host` isn't working for me :(

Reply:
This is necessary so that the container binds to `127.0.0.1`. What error do you get?

Reply:
Actually, `-p 11222:11222` is sufficient.

Reply:
the connection is blocked somewhere. The server starts and binds to the port successfully, but I can't connect (neither with the browser)
without it, it works. And If I use the "zip" version, it works as well. 
On a side note, I would add `--rm` to remove the container when it stops.

Reply:
Hmm not sure why that's the case, but I've removed the flag anyway. 

> On a side note, I would add --rm to remove the container when it stops.

I would rather not add this as I want to keep the command as simple as possible. We're trying to educate users on how to use Infinispan, not `docker|podman|<some-other-container-runtime>`